WCON_format.md

```diff
@@ -377,9 +379,9 @@ In this example, we imagine a directory containing files labelled `filename_0.wc
 ```JSON
 {
     "files":{
-         "this":"_2",
-         "prev":["_1", "_0"],
-         "next":"_3"
+        "this":"_2",
+        "prev":["_1", "_0"],
+        "next":["_3"]
```

Comment:
Regardless of what we decide for brackets overall, here I think it's important to allow them to be dropped since the minimum needed is simply the prev and next item.  Any additional items are just for error-checking.  It's the very next and very previous ones that let you follow the chain, so a reader should only need to say them and ignore the possibility of having a comprehensive list in every file.